uint8 GREET=0
uint8 FAREWELL=1

uint8 command # Should be one of GREET, FAREWELL
uint8 repeat_count # Number of times to repeat the sound
float32 repeat_delay_sec  # Delay between repetitions in seconds